test/features/support/world/world.js

```diff
@@ -0,0 +1,12 @@
+const { setWorldConstructor } = require("cucumber");
```

Comment:
can you explain to me what is the goal for this class?

Reply:
The idea i have for the world it's that it lets me create a class that can exchange information between scenarios. I used it at first because i had only one step file, so for me to update the variables, for example "response" or "query" then i had to create local variable for each scenario(i tried to update only one variable, for example, "response" but it started failing, it didn't get updated when the scenario needed it)   so i ended up with response_1, response_2 , .. etc.

So i use world to persist variables or change variables through the scenarios, and also it helps me to divide the test into a specific test or a common test, like when we test the status code, that is a common test for almost every scenario, with world i can separate them into a common file and makes the steps more readable